source/query/atlas-search.txt

```diff
@@ -0,0 +1,46 @@
+.. _run-atlas-search-query:
+
+========================
+Run Atlas Search Queries
+========================
+
+.. default-domain:: mongodb
+
+For data on the Atlas cluster, if you :ref:`created the Atlas Search index
+<compass-create-search-index>`, you can run Atlas Search
```

Comment:
[suggestion] I would suggest putting the [conditional clause first ](https://www.mongodb.com/docs/meta/style-guide/writing/write-clear-concise-sentences-paragraphs/#use-a-consistent-sentence-structure) re the style guide